src/WebInterface.hs

```diff
@@ -243,7 +266,10 @@ viewPullRequest info pullRequestId pullRequest = do
         (BuildStarted ciUrl)       -> span " | " >> ciLink ciUrl "CI build"
         (BuildFailed (Just ciUrl)) -> span " | " >> ciLink ciUrl "CI build"
         _                          -> pure ()
-    _ -> pure ()
+    Conflicted _ _      -> span "  | " >> span "‼️ conflicted"
```

Comment:
Should this !! be some `:github-emoji-thing:` instead for consistency?

Reply:
That would not work in this case, as this is our internal web interface, `:github-emoji-thing:` is not interpreted as an emoji.

Reply:
Ahh, of course. Silly me